src/services/AttachmentResolver.js

```diff
@@ -249,6 +260,12 @@ export default class AttachmentResolver {
 		return pathNormalize(f)
 	}
 
+	async #getMimeUrl(src) {
+		const headResponse = await axios.head(src)
```

Comment:
When getting the HEAD response, we could actually store more metadata (in particular the filesize) that can be reused in https://github.com/nextcloud/text/blob/0f6805c90210699390f639fa285764d83056b127/src/nodes/ImageView.vue#L46.

My current take would be to register a `metadata` key to the `results` entry from `resolve` above and check at https://github.com/nextcloud/text/blob/0f6805c90210699390f639fa285764d83056b127/src/nodes/ImageView.vue#L315 whether that's set before doing the extra request. What do others think?

Reply:
Done this now.